examples/pytorch/graphsage/experimental/train_dist.py

```diff
@@ -265,8 +266,9 @@ def run(args, device, data):
 def main(args):
     if not args.standalone:
         th.distributed.init_process_group(backend='gloo')
+
+    dgl.distributed.init_rpc(args.ip_config, num_workers=args.num_workers)
```

Comment:
George suggested to use a more general name for this. let's rename it as `initialize`.